decidim-admin/app/forms/decidim/admin/component_form.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+module Decidim
+  module Admin
+    # A form object used to create participatory processes from the admin
+    # dashboard.
+    #
+    class ComponentForm < Decidim::Form
+      include TranslatableAttributes
+
+      mimic :component
+
+      translatable_attribute :name, String
+      attribute :step_id, Integer
```

Comment:
To me, form objects should handle the data from the form, and the form gives Ids. The component is the one who should handle the objects, IMO.